src/AccessibilityInsights.Extensions.AzureDevOps/FileIssue/FileIssueHelpers.cs

```diff
@@ -274,57 +288,7 @@ private static string GetTempFileName(string extension)
             return Path.Combine(GetTempDir(), Path.GetRandomFileName() + extension);
         }
 
-        public static Task<string> AttachScreenshotToIssueAsync(string path, int issueId)
-        {
-            return AzureDevOps.AttachScreenshotToIssue(path, issueId);
-        }
-
-        public static Task<int?> AttachTestResultToIssueAsync(string path, int issueId)
-        {
-            return AzureDevOps.AttachTestResultToIssue(path, issueId);
-        }
-
-        public static Task ConnectAsync(Uri uri, CredentialPromptType prompt)
-        {
-            return AzureDevOps.ConnectToAzureDevOpsAccount(uri, prompt);
-        }
-
-        public static ConnectionCache CreateConnectionCache(string configString)
-        {
-            return new ConnectionCache(configString);
-        }
-
-        public static ConnectionInfo CreateConnectionInfo(Uri serverUri, TeamProject project, AdoTeam team)
-        {
-            return new ConnectionInfo(serverUri, project, team);
-        }
-
-        public static ConnectionInfo CreateConnectionInfo(string configString)
-        {
-            try
-            {
-                return new ConnectionInfo(configString);
```

Comment:
This ctor is only used in unit tests. I'll probably remove it, but likely in a separate PR